addons/mail/static/src/components/attachment_list/attachment_list.scss

```diff
@@ -6,8 +6,11 @@
     display: flex;
     flex-flow: wrap;
     justify-content: flex-start;
+    margin: map-get($spacers, 1);
 }
 
 .o_AttachmentList_attachment {
-    margin: map-get($spacers, 1);
+    margin: map-get($spacers, 1) 0;
+    margin-inline-end: map-get($spacers, 1);
+    max-width: 100%;
```

Comment:
If I understand this is necessary to avoid overflow of long attachment text due to parent being flex? Might be worth a comment.